FSharp.AutoComplete/Program.fs

```diff
@@ -617,6 +641,36 @@ module internal Main =
 
               main state
 
+          | SymbolUse ->
+              let symboluses =
+                  async {
+                      let! symboluse = tyRes.GetSymbol(line, col, lineStr)
+                      if symboluse.IsNone then return None else
+                      let! symboluses = tyRes.GetUsesOfSymbolInFile symboluse.Value.Symbol
+                      return Some {
+                        Name = symboluse.Value.Symbol.DisplayName
+                        Uses =
+                          [ for su in symboluses do
+                              yield { StartLine = su.RangeAlternate.StartLine
+                                      StartColumn = su.RangeAlternate.StartColumn + 1
+                                      EndLine = su.RangeAlternate.EndLine
+                                      EndColumn = su.RangeAlternate.EndColumn + 1
```

Comment:
I agree with you that it is very confusing. If you look at the test output I linked in the last comment I found that `GetDeclaration` returns 0-based. So does the one that returns all the declarations in the file, but that isn't really used at the moment.